src/contributing_to_atmosphere/contributing_to_atmosphere.md

```diff
@@ -14,6 +15,55 @@ Atmosphere or 'atmo' can refer to several different things:
+        2.  Put the ticket into "**Review**" step and assign to a project lead. (See Reviewing Code section for more details)
+        3.  Upon merge, a project lead will move the ticket to **Resolved** and assigned to QA for final approval/confirmation. Notes should be included that point to the new tests/documentation as appropriate.
+
+### Using github to work on an issue/bug/feature
+1.  **Fork the repo you intend to work on**
+    1.  If this is your first time contributing to a core-services repository, you will want to create a new fork.<span style="color: rgb(34,34,34);"> </span>Forking a repository<span style="color: rgb(34,34,34);"> allows you to freely experiment with changes without affecting the original project.</span>
+    2.  <span style="color: rgb(34,34,34);">Github's documentation for working with forks is [available here](https://help.github.com/articles/fork-a-repo/).</span>
+2.  **<span style="color: rgb(34,34,34);">Creating a feature branch</span>**
+    1.  Before starting work on a new feature or bugfix, you should create a new branch. Creating branches for your work allows you to isolate your changes when they are ready for others to review. It also gives you the ability to change to another branch, to work on separate fixes or features as necessary.
+3.  **Submitting a Pull Request**
+    1.  It's a good idea to make pull requests early on. A pull request represents the start of a discussion, and doesn't necessarily need to be the final, finished submission.
+    2.  It's also useful to remember that if you have an outstanding pull request then pushing new commits to your GitHub repo will also automatically update the pull requests.
+    3.  **Specific to Core-services:** Pull requests will be reviewed **after** the "Check list" in the Pull Request template has been completed, this should include new tests and documentation!
+    4.  GitHub's documentation for working on pull requests is [available here](https://help.github.com/articles/using-pull-requests).
+
+
+#### Pull Request Templates
+
+The pull request template used by core-services can be found on the github of Atmosphere or Troposphere in .**/PULL_REQUEST_TEMPLATE**
+
+> <span style="color: rgb(255,0,0);">**NOTE:** Proposals for those PR templates can be found here, and can be updated prior to merge: [<span style="color: rgb(255,0,0);">https://github.com/cyverse/troposphere/pull/536</span>](https://github.com/cyverse/troposphere/pull/536) and [<span style="color: rgb(255,0,0);">https://github.com/cyverse/atmosphere/pull/250</span>](https://github.com/cyverse/atmosphere/pull/250)</span>
+
+See this article for including a pull request template in your repository: [https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/)
```

Comment:
Are we writing for an audience of maintainers or contributors here? Should we set this for all of our repos so that contributors reading this don't need to worry about whether the template is included?